integration-tests/emulated/tests/collectives/collectives-polkadot/src/tests/fellowship_treasury.rs

```diff
@@ -0,0 +1,233 @@
+use xcm_executor::traits::ConvertLocation;
+
+// Fund Fellowship Treasury from Polkadot Treasury and spend from Fellowship Treasury.
+#[test]
+fn fellowship_treasury_spend() {
+	// initial treasury balance on Asset Hub in DOTs.
+	let treasury_balance = 20_000_000 * UNITS;
+	// target fellowship balance on Asset Hub in DOTs.
+	let fellowship_treasury_balance = 1_000_000 * UNITS;
+	// fellowship first spend balance in DOTs.
+	let fellowship_spend_balance = 10_000 * UNITS;
+
+	let init_alice_balance = AssetHubPolkadot::execute_with(|| {
+		<<AssetHubPolkadot as AssetHubPolkadotPallet>::Balances as Inspect<_>>::balance(
+			&Polkadot::account_id_of(ALICE),
+		)
+	});
+
+	Polkadot::execute_with(|| {
+		type RuntimeEvent = <Polkadot as Chain>::RuntimeEvent;
+		type RuntimeCall = <Polkadot as Chain>::RuntimeCall;
+		type Runtime = <Polkadot as Chain>::Runtime;
+		type Balances = <Polkadot as PolkadotPallet>::Balances;
+		type Treasury = <Polkadot as PolkadotPallet>::Treasury;
+
+		// Fund Treasury account on Asset Hub with DOTs.
+
+		let root = <Polkadot as Chain>::RuntimeOrigin::root();
+		let treasury_account = Treasury::account_id();
+
+		// Mint assets to Treasury account on Relay Chain.
+		assert_ok!(Balances::force_set_balance(
+			root.clone(),
+			treasury_account.clone().into(),
+			treasury_balance * 2,
+		));
+
+		let native_asset = Location::here();
+		let asset_hub_location: Location = [Parachain(1000)].into();
+		let treasury_location_on_asset_hub: Location = (Parent, PalletInstance(19)).into();
+
+		let teleport_call = RuntimeCall::Utility(pallet_utility::Call::<Runtime>::dispatch_as {
+			as_origin: bx!(OriginCaller::system(RawOrigin::Signed(treasury_account))),
+			call: bx!(RuntimeCall::XcmPallet(pallet_xcm::Call::<Runtime>::teleport_assets {
+				dest: bx!(VersionedLocation::V4(asset_hub_location.clone())),
+				beneficiary: bx!(VersionedLocation::V4(treasury_location_on_asset_hub)),
+				assets: bx!(VersionedAssets::V4(
+					Asset { id: native_asset.clone().into(), fun: treasury_balance.into() }.into()
+				)),
+				fee_asset_item: 0,
+			})),
+		});
+
+		// Dispatched from Root to `despatch_as` `Signed(treasury_account)`.
+		assert_ok!(teleport_call.dispatch(root));
+
+		assert_expected_events!(
+			Polkadot,
+			vec![
+				RuntimeEvent::XcmPallet(pallet_xcm::Event::Sent { .. }) => {},
+			]
+		);
+	});
+
+	Polkadot::execute_with(|| {
+		type RuntimeEvent = <Polkadot as Chain>::RuntimeEvent;
+		type RuntimeCall = <Polkadot as Chain>::RuntimeCall;
+		type RuntimeOrigin = <Polkadot as Chain>::RuntimeOrigin;
+		type Runtime = <Polkadot as Chain>::Runtime;
+		type Treasury = <Polkadot as PolkadotPallet>::Treasury;
+
+		// Fund Fellowship Treasury from Polkadot Treasury.
+
+		let treasury_origin: RuntimeOrigin =
+			polkadot_runtime::governance::pallet_custom_origins::Origin::Treasurer.into();
+		let fellowship_treasury_location: Location =
+			Location::new(1, [Parachain(1001), PalletInstance(65)]);
```

Comment:
Yeah I definitely think we should try to access the actual pallet's index for these locations

Reply:
I do not have that strong opinion about this. When I use literals, I want this test fail if for some reason pallet's index changes.